addons/l10n_ro/data/account_tax_data.xml

```diff
@@ -1145,7 +1145,30 @@
     <record id="tvati_extra" model="account.tax.template">
         <field name="chart_template_id" ref="ro_chart_template"/>
         <field name="sequence">35</field>
-        <field name="name">TVA Export</field>
+        <field name="name">TVA Export Bunuri</field>
```

Comment:
Normally we add this to have it in the fiscal positions...

Reply:
Yes :sweat_smile:

Reply:
still missing ?

Reply:
Thanks.